src/styles/main.scss

```
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+$countDay: 7;
+$sizeDay: 100px;
+$days:
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $sizeDay * $countDay;
+  gap: 1px;
+  margin: 0 auto;
+  padding: 25px;
+  cursor: pointer;
+
+  @for $length from 28 through 31 {
+    &--month-length-#{$length} &__day:nth-child(n+#{$length + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $index in $days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: calc(84.9px * #{$index} + #{$index}px);
```

use integers for pixel values
```suggestion
      margin-left: calc(84.9px * #{$index} + #{$index}px);
```